src/explore-education-statistics-admin/src/modules/find-statistics/components/Comments.module.scss

```diff
@@ -35,5 +35,10 @@
 }
 
 .top {
+  height: auto;
   z-index: 1000;
 }
+
+.minCommentHeight {
```

Comment:
The style class name should relate to the component it is styling, not the containing styles